### PR TITLE
Add limits to horizontal scaler walkthrough

### LINKED ISSUE #15600: Issue with k8s.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/
**This is a Bug Report**

**Problem:**
Autoscaing doesn't work if you don't specify **cpu** **limits** for deployment.
`kubectl run php-apache --image=k8s.gcr.io/hpa-example --requests=cpu=200m --expose --port=80`
**Proposed Solution:**
Add `--limits=cpu=500m` parameter to deploy command:
`kubectl run php-apache --image=k8s.gcr.io/hpa-example --requests=cpu=200m --limits=cpu=500m --expose --port=80`
**Page to Update:**
https://kubernetes.io/...

### PR DESCRIPTION
**Pull request by thecrudge**
Resolves #15600

### CODE REVIEW

File: content/en/examples/pods/config/redis-pod.yaml [hunk 1]
```diff
@@ -15,10 +15,12 @@ spec:
...
         cpu: "0.1"
     volumeMounts:
-    - mountPath: /redis-master-data
+    - mountPath: /data
       name: data
-    - mountPath: /redis-master
+    - mountPath: /usr/local/etc/redis
       name: config
+    args:
+    - /usr/loca/etc/redis/redis.conf
```
Comment (sftim) [inline content/en/examples/pods/config/redis-pod.yaml]: Is `/usr/loca/etc/redis/redis.conf` a typo? (“loca”)

**Comment by thecrudge** [comment]
That should have not been apart of this commit

File: content/en/examples/pods/config/redis-pod.yaml (hunk 1 above)
Comment (thecrudge) [reply content/en/examples/pods/config/redis-pod.yaml]: Should not have been part of this commit. Removed.

**Pull request merged**